# HG changeset patch
# User Rik <rik@octave.org>
# Date 1454256659 28800
#      Sun Jan 31 08:10:59 2016 -0800
# Node ID a610c00dc55696031345e2bb8cb4d5b6d94d9a0e
# Parent  2d22bd38deac48f4da032f3dc42d405dd8343842
build: Remove stray '@' from rule to built octave.html.

* doc/module.mk: Remove stray '@' from rule to built octave.html.

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -301,17 +301,17 @@ doc/interpreter/octave.pdf: doc/interpre
 
 doc/interpreter/octave.html: $(OCTAVE_HTML_STAMP)
 
 $(OCTAVE_HTML_STAMP): doc/interpreter/octave.texi $(srcdir)/doc/interpreter/version-octave.texi | doc/interpreter/$(am__dirstamp)
 	$(AM_V_MAKEINFO)rm -rf $(OCTAVE_HTML_DIR)
 	$(AM_V_at)if $(MAKEINFOHTML) $(AM_MAKEINFOHTMLFLAGS) $(MAKEINFOFLAGS) -I doc/interpreter -I $(abs_top_srcdir)/doc/interpreter \
 	 -o $(OCTAVE_HTML_TMP_DIR) `test -f 'doc/interpreter/octave.texi' || echo '$(abs_top_srcdir)/'`doc/interpreter/octave.texi; \
 	then \
-	  rm -rf $(OCTAVE_HTML_DIR)@ && \
+	  rm -rf $(OCTAVE_HTML_DIR) && \
 	  mv $(OCTAVE_HTML_TMP_DIR) $(OCTAVE_HTML_DIR) && \
 	  touch $@; \
 	else \
 	  rm -rf $(OCTAVE_HTML_TMP_DIR); exit 1; \
 	fi
 
 $(HTMLDIR_IMAGES) : doc/interpreter/octave.html/%.png: doc/interpreter/%.png $(OCTAVE_HTML_STAMP)
 	$(AM_V_GEN)cp $< $@
